SpatialGDK/Source/SpatialGDK/Public/Interop/Connection/SpatialEventTracer.h

```diff
@@ -132,13 +132,12 @@ FSpatialGDKSpanId SpatialEventTracer::TraceEvent(const char* EventType, const ch
 	// This would allow for sampling dependent on trace event data.
 	Trace_Event Event = { nullptr, 0, Message, EventType, nullptr };
 
-	if (!Trace_EventTracer_ShouldSampleSpan(EventTracer, Causes, NumCauses, &Event))
+	FSpatialGDKSpanId TraceSpanId;
```

Comment:
I'm guessing default-constructing a `FSpatialGDKSpanId` gives it a null value (such that line 141 assigns a pointer to a null span ID value?

Reply:
Yah!